go.mod

```diff
@@ -171,3 +171,7 @@ replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.15.5+incompatible
 
 replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
+
+replace github.com/spf13/cobra v0.0.0-00000000000000-c46add8a6528 => github.com/spf13/cobra v0.0.0-20170711120833-c46add8a6528
```

Comment:
So why the Cobra replace now? If anything we should try to reduce the amount of replace statements needed instead of increasing them.  How is this related to the xerror problem?

Reply:
I had to because Go 1.13 is more strict about [version validations](https://golang.org/doc/go1.13#version-validation).

Reply:
Yeah I had a little look a couple of weeks back and ran into a number of issues with dependencies and mod `replace ->`.  We might need to hold this change and do a lot more validation upgrading to 1.13, which includes upgrading our builder images so our pipelines build with 1.13 and that will impact other repos too.